docs/beta-scenarios.md

```diff
@@ -0,0 +1,103 @@
+4. Select the connector you downloaded from the Extension Gallery and fill out the information in the dialog box once prompted.
+5. Create an identical data source as the one you are about to replace (same tables and joins, or customer SQL).
+6. Navigate to a Sheet within the Workbook (or create a new one). You should have two data sources now.
+        DB - New Connector
+        DB - ODBC
+7. Replace the datasource <code>Data > Replace Data Source...</code>
+8. Interact with the Sheets and Dashboard as you had previously to verify the performance and the accuracy of the data.
+
+## Test Scenario 3 - Publishing that Workbook to Tableau Server
+After completing test scenarios 1 and 2, the next step is to publish your Workbook with the new datasource to Tableau Server
+
+1. From the Workbook in Test Scenario 2 select <code>Server > Publish Workbook</code> from the main menu.
+2. Make sure you are already logged in to the server you published the connector to in Test Scenario 1.
+3. Select a relevant Project and Name for your Workbook and click Publish.
+4. View the Workbook on Tableau Server and verify the Workbook performance and accuracy.
+
+## Test Scenario 4 - Replacing a published datasource for a Workbook on Tableau Server
+After completing the previous steps, verify that you can replace a published datasource from Tableau Server for a Workbook already in use. This is effectively a combination of Test Scenario 2 and 3 except performed on an existing Workbook connected to an existing published datasource. We recommend keeping a local copy with the original connector available for comparison/roll-back in case of an issue.
+
+1. Navigate to Tableau Server and download a Workbook connected to a published datasource.
+2. Create a new datasource <code>Data > New Data</code>, and select the new connector.
+3. Select the connector you downloaded from the Extension Gallery and fill out the information in the dialog box once prompted.
+4. Create an identical data source as the one you are about to replace (same tables and joins, or customer SQL).
+Navigate to a Sheet within the Workbook (or create a new one). You should have two data sources:
+   * DB - New Connector
+   * DB - ODBC
+5. Replace the datasource (Data > Replace Data Source...)
+Interact with the Sheets and Dashboard as you had previously to verify the performance and the accuracy of the data.
+6. From the Workbook select <code>Server > Publish Workbook</code> from the main menu.
+7. Make sure you are already logged in to the server you published the connector to in Test Scenario 1.
+8. Keep the existing Project and Name for your Workbook and click Publish.
+9. Click Okay to overwrite the existing Workbook when prompted.
+10. View the Workbook on Tableau Server and once again verify the Workbook performance and accuracy.
+
+<br>
+
+*Thank you for validating your connector through the Beta Test Scenarios!*
```

Comment:
Nit: this isn't their connector they are testing, but someone else's they are making.